docs/usage/configuration-options.md

```diff
@@ -463,6 +463,128 @@ Make sure that version is a pinned version of course, as otherwise it won't be v
 
 For `sbt` note that Renovate will update the version string only for packages that have the version string in their project's `built.sbt` file.
 
+## bumpVersions
+
+The `bumpVersions` option allows Renovate to update semantic version strings in your code when dependencies are updated.
+This is useful for files or version fields that Renovate does not natively support or for custom versioning needs.
+
+The option is an array of rules, each specifying how and where to bump versions. Each rule includes the following fields:
+
+- `filePatterns`: A list of regex patterns to match file names. These patterns follow Renovate's [string pattern matching syntax](./string-pattern-matching.md). Templates can also be used for dynamic patterns.
+- `matchStrings`: An array of regex patterns to locate version strings within the matched files. Any of the regexes can match the content. Each pattern must include a named capture group `version` to extract the version string.
+- `bumpType`: Specifies the type of version bump which defaults to `patch`. Supported values are:
+  - `prerelease`
+  - `patch`
+  - `minor`
+  - `major`
+    This field supports templates for conditional logic.
+- `name` (optional): A descriptive name for the rule, which is used in logs for easier identification.
+
+<!-- prettier-ignore -->
+!!! tip
+    You can use templates in `filePatternTemplates`, `bumpType`, and `matchStrings`.
+    This way you can leverage the power of Renovate's templating engine to change based on the context of the upgrade.
+
+Here is an example of a `bumpVersions` configuration:
+
+```json
+{
+  "bumpVersions": [
+    {
+      "name": "Updating release version file",
+      "filePatterns": [".release-version"],
+      "bumpType": "minor",
+      "matchStrings": ["^(?<version>.+)$"]
+    }
+  ]
+}
+```
+
+In this example:
+
+- Renovate scans files named `.release-version`.
+- It matches the entire file content as the version string.
+- It bumps the version to the next minor release.
+
+**Conditional Bumping with `packageRules`:**
+
+You can use `packageRules` to apply `bumpVersions` conditionally.
+For example, to bump versions only for updates in the `charts/` directory:
+
+```json
+{
+  "packageRules": [
+    {
+      "matchFileNames": ["charts/**"],
+      "bumpVersions": {
+        "filePatterns": "{{packageFileDir}}/Chart.{yaml,yml}",
```

Comment:
This should be a list

Reply:
This has been already corrected in the docs